mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -406,26 +440,32 @@ def _compute_moc_climo_postprocess(self):  # {{{
             # rename some variables for convenience
             annualClimatology = annualClimatology.rename(
                     {'timeMonthly_avg_normalVelocity': 'avgNormalVelocity',
-                     'timeMonthly_avg_vertVelocityTop': 'avgVertVelocityTop'})
+                     'timeMonthly_avg_vertVelocityTop': 'avgVertVelocityTop',
+                     'timeMonthly_avg_layerThickness': 'avgLayerThcikness'})
             annualClimatology = annualClimatology.isel(Time=0)
 
             # Convert to numpy arrays
             # (can result in a memory error for large array size)
             horizontalVel = annualClimatology.avgNormalVelocity.values
             verticalVel = annualClimatology.avgVertVelocityTop.values
-            velArea = verticalVel * areaCell[:, np.newaxis]
+            velArea = verticalVel * self.areaCell[:, np.newaxis]
+
+            layerThicknessCell = annualClimatology.avgLayerThcikness.values
+            layerThicknessEdge = \
```

Comment:
I compute layer thickness at edges by averaging layer thickness from adjacent cells (exactly as done in MPAS-Ocean).